### PR TITLE
PN-349: Add ability to mark matches as notified or un-notified in the matches table

### PR DESCRIPTION
**Pull request by veronikaslc**
- Improvement: Make 'Mark notified' button visible to both admins and users

### CODE REVIEW

**Comment by sashaandjic** [comment]
I would like to have option to also "UNMARK CONTACTED" in case I click on the button by mistake.
Can we toggle MARK/UNMARK?

**Comment by veronikaslc** [comment]
We discussed "UNMARK CONTACTED" button with @allasm, he suggested that we do not need this option.

**Comment by veronikaslc** [comment]
Added "Mark un-contacted" button.

**Comment by sashaandjic** [comment]
Click/unclick on MARK NOTIFIED (UNMARK NOTIFIED) does not work. Once I click on it , I need to refresh the screen to be able to unmark it.
I think gray MARK NOTIFIED button may look better than orange.

**Comment by sashaandjic** [comment]
Overal label: Contact
Sublabels: Status; Update

Reword "MARK NOTIFIED" to MARK AS CONTACTED ( drop AS if not enought room)

File: matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js [hunk 1]
```diff
@@ -961,42 +965,61 @@ var PhenoTips = (function (PhenoTips) {
         return td;
     },
 
-    _getNotified : function(record)
+    _getContact : function(record)
     {
-        var td = '<td style="text-align: center" name="contacted">';
+        var td = '<td style="text-align: center" name="contact">';
```
Comment (allasm) [inline matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js]: it is better to have a class name and not embedded CSS

File: matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js [hunk 2]
```diff
@@ -961,42 +965,61 @@ var PhenoTips = (function (PhenoTips) {
...
+    },
+
+    _getContactButton : function(record)
+    {
+        var button = '';
         var accessAboveEdit = record.matched.access == "edit"   || record.reference.access == "edit"
                            || record.matched.access == "owner"  || record.reference.access == "owner"
                            || record.matched.access == "manage" || record.reference.access == "manage";
-        if (!this._isAdmin && accessAboveEdit && (!record.matched.pubmedIDs || record.matched.pubmedIDs.size() == 0) || (!record.reference.pubmedIDs || record.reference.pubmedIDs.size() == 0)) {
+        var oneCaseOrMoreUnsolved = (!record.matched.pubmedIDs || record.matched.pubmedIDs.size() == 0) || (!record.reference.pubmedIDs || record.reference.pubmedIDs.size() == 0);
```
Comment (allasm) [inline matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js]: Not directly related to this issue, but seems to be a regression where we allow contacting patient that are solved

**Comment by veronikaslc** [comment]
Rebased to resolve conflict with master

**Comment by sashaandjic** [comment]
If case is solved and there is no PubMedID, we should allow contact, shouldn't we? 
screen shot 2018-11-20 at 10 54 47 am

**Pull request merged**